.github/workflows/component_packages.yml

```diff
@@ -70,6 +71,14 @@ jobs:
         run: |
           cargo install --locked cargo-zigbuild --force
 
+      - name: Install Windows cross-compilation dependencies
+        run: |
+          sudo apt-get install -y llvm
```

Comment:
I also tried the action `KyleMayes/install-llvm-action` but when installing that, the runner runs out of space.